Makefile

```diff
@@ -181,7 +181,7 @@ docker-build: test
 # Build docker image with buildx
 # Experimental docker feature to build cross platform multi-architecture docker images
 # https://docs.docker.com/buildx/working-with-buildx/
-docker-buildx-dev:
+docker-buildx-dev: test
```

Comment:
add `test` for `docker-buildx-release` also

Reply:
Addressed 👍